x-pack/plugins/security_solution/server/lib/telemetry/types.ts

```diff
@@ -7,7 +7,7 @@
 
 import type { AlertEvent, ResolverNode, SafeResolverEvent } from '../../../common/endpoint/types';
 
-type BaseSearchTypes = string | number | boolean | object;
+type BaseSearchTypes = string | number | boolean | object | null;
```

Comment:
making timelineId and timelineTitle nullable requires changes in this typing as well